real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/simulation/ControllerTester.java

```diff
@@ -0,0 +1,113 @@
+package com.roboclub.robobuggy.simulation;
+
+import Jama.Matrix;
+import com.roboclub.robobuggy.messages.DriveControlMessage;
+import com.roboclub.robobuggy.messages.GPSPoseMessage;
+import com.roboclub.robobuggy.nodes.baseNodes.BuggyBaseNode;
+import com.roboclub.robobuggy.nodes.baseNodes.PeriodicNode;
+import com.roboclub.robobuggy.nodes.localizers.LocTuple;
+import com.roboclub.robobuggy.nodes.localizers.LocalizerUtil;
+import com.roboclub.robobuggy.nodes.localizers.UTMTuple;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.ros.Publisher;
+import com.roboclub.robobuggy.ros.Subscriber;
+
+import java.util.Date;
+
+/**
+ * Created by vivaanbahl on 1/25/17.
+ *
+ * adapted from @babraham's MATLAB script found in offline/controller
+ */
+public class ControllerTester extends PeriodicNode {
+
+    // run every 10 ms
```

Comment:
This is a simulator, so I guess this is okay? Unless we want these parameters in a config file